docs/content/integrations/snowflake/reference.mdx

```diff
@@ -110,6 +111,113 @@ When Dagster materializes `sepal_data` and loads the `iris_dataset` asset using
 
 ---
 
+## Storing partitioned assets
+
+The Snowflake I/O manager supports storing and loading partitioned data. In order to correctly store and load data from the Snowflake table, the Snowflake I/O manager needs to know which column contains the data defining the partition bounds. The Snowflake I/O manager uses this information to construct the correct queries to select or replace the data. In the following sections, we describe how the I/O manager constructs these queries for different types of partitions.
+
+### Storing static partitioned assets
+
+In order to store static partitioned assets in Snowflake, you must specify `partition_expr` metadata on the asset to tell the Snowflake I/O manager which column contains the partition data:
```

Comment:
❓ How does one format what goes into `partition_expr`? Ex: Since this is Snowflake, do column names need to be uppercase?

Reply:
I dont think snowflake is case sensitive. I could link out tho this very dense docs page of snowflake sql queries https://docs.snowflake.com/en/sql-reference/constructs

Reply:
Spitballing on phrasing:

> `partition_expr` accepts valid Snowflake SQL queries. Refer to the [Snowflake documentation](link) for info on Snowflake's query syntax.

Thoughts?

Reply:
i don't want to imply that they can put any sql query into "partition_expr" since we're injecting it into another query and that could then be not-valid.  but i think I have an idea where to put this. updating the guide now

Reply:
updated! let me know what you think. I added this sentence in 
> When the `partition_expr` value is injected into this statement, the resulting SQL query must follow Snowflake's SQL syntax. Refer to the [Snowflake documentation](https://docs.snowflake.com/en/sql-reference/constructs) for more information.

Reply:
I like it!